src/main/java/mate/academy/spring/model/Role.java

```diff
@@ -0,0 +1,40 @@
+package mate.academy.spring.model;
+
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "roles")
+public class Role {
```

Comment:
![image](https://github.com/mate-academy/jv-spring-security-part-2/assets/20326858/9cb8a149-27cb-4081-a072-b92b47c8ac70)

Reply:
not fixed